models/project/project.go

```diff
@@ -217,15 +217,10 @@ func CountProjects(ctx context.Context, opts SearchOptions) (int64, error) {
 
 // FindProjects returns a list of all projects that have been created in the repository
 func FindProjects(ctx context.Context, opts SearchOptions) ([]*Project, int64, error) {
-	e := db.GetEngine(ctx)
+	e := db.GetEngine(ctx).Table("project")
```

Comment:
IIRC, we don't really need this, `FindAndCount` will detect the table name.

Reply:
‘FindAndCount’ is not directly exposed here, so I added ‘Table’. 
It seems that only an xorm session can use ‘FindAndCount’ directly, but db.GetEngine returns ‘Engine’ which is not an xorm session here. Is it possible to get an xorm session here without using ‘Table’?

Reply:
Maybe:

```go
func FindProjects(ctx context.Context, opts SearchOptions) ([]*Project, int64, error) {
	e := db.GetEngine(ctx).Where(opts.toConds())
	if opts.Page > 0 {
		e.Asc("created_unix")
	}
	
	projects := make([]*Project, 0, setting.UI.IssuePagingNum)
	count, err := e.FindAndCount(&projects)
	return projects, count, err
}
```

Reply:
`db.GetEngine` won't return a session but `Where` will. I missed this. Thanks a lot.